python/ray/dag/compiled_dag_node.py

```diff
@@ -990,17 +1037,21 @@ def _preprocess(self) -> None:
                 if actors not in actors_to_nccl_group_id:
                     actors_to_nccl_group_id[actors] = nccl_group_id
 
-        # If a NCCL group for P2P actors is not initialized, initialize and cache
-        # the NCCL group ID.
-        if nccl_actors_p2p and self._nccl_group_id_p2p is None:
-            actors = frozenset(nccl_actors_p2p)
+        # If no default NCCL group for P2P actors was specified:
+        # 1. Try to reuse another NCCL group in the DAG.
+        # 2. Create a `_NcclGroup` if we cannot reuse.
```

Comment:
Can you make these steps consistent with the docstring?

Reply:
Shortened this comment and kept the details in the docstring.